runtime/service.go

```diff
@@ -435,14 +441,21 @@ func (s *service) waitVMReady() error {
 func (s *service) CreateVM(requestCtx context.Context, request *proto.CreateVMRequest) (*proto.CreateVMResponse, error) {
 	defer logPanicAndDie(s.logger)
 
+	timeout := defaultCreateVMTimeout
+	if request.TimeoutSeconds > 0 {
+		timeout = time.Duration(request.TimeoutSeconds) * time.Second
```

Comment:
Is there a sensible maximum we'd want to enforce for TimeoutSeconds?

Reply:
I assume container clients are containerd operators. People won't and shouldn't expose containerd to the internet. Because of that, if clients want to wait 30 minutes or even one hour, it is their choice.